### ISSUE TITLE
Add JSON Schema

### ISSUE THREAD

**Issue by tiero** [premAI-io/prem-registry#82]

**Comment by casperdcl** [comment]
Where will be the source of truth? We also have:

- https://github.com/premAI-io/prem-daemon/blob/main/app/schemas.py
- https://github.com/premAI-io/prem-app/blob/main/src/modules/service/types.ts

which are already a bit out of sync.

**Comment by tiero** [comment]
As we are deprecating the current alpha daemon, we agreed to not have any service to "enrich" the list of the manifest(s) with more data at runtime, such as service status etc.. 

Registry will be deterministic and MUST have all the keys all the time, no optional keys, only nullable values.

**Comment by casperdcl** [comment]
yes but my point is the links above independently redefine the schema. Ideally we want them to use the same schema.

**Comment by tiero** [comment]
The SCHEMA imho is something we want to provide to who package service to check at bike time their manifest.json is correct.

Clients such as the app, may use it to validate at runtime when sideloading manifest outside our registry or to load their own hosted registry. 

The daemon may want to do it as well, but it should not add their own types that are the API response /v1/services etc as we are deprecating that soon
